packages/rum-core/src/domain/rumEventsCollection/view/trackInitialViewTimings.ts

```diff
@@ -75,14 +77,16 @@ export function trackNavigationTimings(lifeCycle: LifeCycle, callback: (timings:
 
 export function trackFirstContentfulPaintTiming(lifeCycle: LifeCycle, callback: (fcpTiming: RelativeTime) => void) {
   const firstHidden = trackFirstHidden()
-  const { unsubscribe: stop } = lifeCycle.subscribe(LifeCycleEventType.PERFORMANCE_ENTRY_COLLECTED, (entry) => {
-    if (
-      entry.entryType === 'paint' &&
-      entry.name === 'first-contentful-paint' &&
-      entry.startTime < firstHidden.timeStamp &&
-      entry.startTime < TIMING_MAXIMUM_DELAY
-    ) {
-      callback(entry.startTime)
+  const { unsubscribe: stop } = lifeCycle.subscribe(LifeCycleEventType.PERFORMANCE_ENTRIES_COLLECTED, (entries) => {
+    for (const entry of entries) {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏there is only one `first-contentful-paint`, so maybe use something like `entries.find(...)` instead of iterating over all entries?